src/read.c

```diff
@@ -179,7 +178,7 @@ static void ReadFuncCallOption(ScannerState * s, TypSymbolSet follow)
     }
     else if (s->Symbol == S_LPAREN) {
         Match(s, S_LPAREN, "(", S_COMMA | follow);
-        ReadExpr(s, follow, 'r');
+        ReadExpr(rs, follow, 'r');
```

Comment:
Aaaand we also see a lot of  these: instead of passing a `ScannerState` named `s`, we pass on a `ReaderState` called `rs`.